net-mgmt/net-snmp/src/opnsense/mvc/app/controllers/OPNsense/Netsnmp/forms/general.xml

```diff
@@ -47,6 +47,24 @@
         <style>tokenize</style>
         <type>select_multiple</type>
         <allownew>true</allownew>
-        <help>Set the IP addresses the service should listen to.</help>
+        <help>Set the IP addresses the service should listen to. For Zabbix Proxy use LAN IP.</help>
```

Comment:
Please revert this one. This would be something like a documentation example, but I'd guess most of the snmp users dont use Zabbix

Reply:
done also other helps where I wrote zabbix parameters